cross/c_icap/Makefile

```diff
@@ -1,8 +1,8 @@
 PKG_NAME = c_icap
-PKG_VERS = 0.2.2
+PKG_VERS = 0.3.5
 PKG_EXT = tar.gz
 PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
-PKG_DIST_SITE = http://downloads.sourceforge.net/project/c-icap/c-icap/0.2.x
+PKG_DIST_SITE = http://downloads.sourceforge.net/project/c-icap/c-icap/0.3.x
 PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
 
```

Comment:
i have always the same error : 
In file included from include/service.h:26,
                 from include/request.h:25,
                 from acl.c:22:
include/ci_threads.h:47: error: expected ';', ',' or ')' before '*' token
include/ci_threads.h:48: error: expected ';', ',' or ')' before '*' token
In file included from include/request.h:25,
                 from acl.c:22:
include/service.h:77: error: expected specifier-qualifier-list before 'pthread_rwlock_t'


i thing that the gcc compiler is too old, 
in x86/64 4.3 gcc version 4.2

i haven't found a workaround to fix compilation error :s